config/routes.rb

```diff
@@ -18,6 +18,8 @@
       get "/names", to: "names#show"
     end
 
+    resources :saved_pages, only: %i[index show create destroy], param: :page_path
```

Comment:
Nice one, I didn't know this one: https://guides.rubyonrails.org/routing.html#overriding-named-route-parameters

Reply:
Yeah I thought that was a cool rails feature too.
The nice bit is, if you run `bundle exec rails routes` it gives you a good print out, and makes clear that we're looking for a `:page_path` instead of a default `:id`!

Reply:
Can we maybe clarify this bit in the commit message?
```
Also we're not using an ID here, but instead uniquely identifying a
resource by a page path. So lets name that here.
```

Reply:
amended!